crates/circuit/src/operations.rs

```diff
@@ -1903,13 +1892,13 @@ impl Operation for StandardGate {
                         py,
                         2,
                         [
-                            (Self::RX, smallvec![Param::Float(PI / 2.)], q0.clone()),
-                            (Self::RX, smallvec![Param::Float(PI / 2.)], q1.clone()),
+                            (Self::SXdg, smallvec![], q0.clone()),
+                            (Self::SXdg, smallvec![], q1.clone()),
                             (Self::CX, smallvec![], q0_q1.clone()),
                             (Self::RZ, smallvec![theta.clone()], q1.clone()),
                             (Self::CX, smallvec![], q0_q1),
-                            (Self::RX, smallvec![Param::Float(-PI / 2.)], q0),
-                            (Self::RX, smallvec![Param::Float(-PI / 2.)], q1),
+                            (Self::SX, smallvec![], q0),
+                            (Self::SX, smallvec![], q1),
```

Comment:
For a very attentive reviewer: RX and SXdg gates differ by a global phase, however all the 4 global phase contributions cancel each other out.